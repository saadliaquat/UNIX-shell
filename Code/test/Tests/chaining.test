ls | grep .json && ls -al | grep total | cut -d ' ' -f 2
cat config.json | grep -oE "easy|medium|advanced" | sort | uniq -c | sort -n || find Tests/ -maxdepth 1 -type f | xargs wc -l | sort -n
false || echo "This should be printed" || echo "This should not be printed"
ls && false || echo "doit" && false && echo "wonâ€™t work" || echo "works" ; echo "done"
false || echo "This should be printed" || echo "This should not be printed"
echo "O" ; echo p ; echo "e" ; echo r ; echo "a" ; echo t ; echo "i" ; echo o ; echo n ; echo "s"