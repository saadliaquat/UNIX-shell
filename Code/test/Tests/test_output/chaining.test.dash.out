config.json
config.json.log
16
      3 advanced
      3 easy
      3 medium
This should be printed
Tests
config.json
config.json.log
find.log
names.txt
test.py
doit
works
done
This should be printed
O
p
e
r
a
t
i
o
n
s
